### PR TITLE
Fix invalid references

### PR DESCRIPTION
**Pull request by setzamora**

### CODE REVIEW

File: zap-backup-test.py [hunk 1]
```diff
@@ -30,42 +30,42 @@ def main(argv):
...
+   print('Loaded: %s' % str(time.strftime('%H:%M:%S', time.gmtime(int(rec_time)))))
 
    rec_mcount = zap.core.number_of_messages()
    rec_acount = zap.core.number_of_alerts()
    if (old_mcount == rec_mcount):
-     print 'PASS: msg counts match'
+     print('PASS: msg counts match')
    else:
-     print 'FAIL: msg counts differ - original: %s recovered: %s' % old_mcount, new_mcount
+     print('FAIL: msg counts differ - original: %s recovered: %s' % old_mcount, new_mcount)
```
Comment (thc202) [inline zap-backup-test.py]: `new_mcount` → `rec_mcount`

File: zap-backup-test.py [hunk 2]
```diff
@@ -30,42 +30,42 @@ def main(argv):
...
    if (old_acount == rec_acount):
-     print 'PASS: alert counts match'
+     print('PASS: alert counts match')
    else:
-     print 'FAIL: alert counts differ - original: %s recovered: %s' % old_acount, new_acount
+     print('FAIL: alert counts differ - original: %s recovered: %s' % old_acount, new_mcount)
```
Comment (thc202) [inline zap-backup-test.py]: `new_mcount` → `rec_acount`

**Comment by thc202** [comment]
I suggest splitting the update of the print statements to work with Python 3 from the invalid references, easier to review and for the history.

**Comment by setzamora** [comment]
> I suggest splitting (i.e. another commit) the update of the print statements to work with Python 3 from the invalid references, easier to review and for the history.

Thanks @thc202. I'll send another PR for the migration to Python 3.

**Comment by thc202** [comment]
Thanks! (Sorry for losing track of these changes.)

**Pull request merged**